Ukoly/DU_10/template.html

```diff
@@ -0,0 +1,13 @@
+<html>
+  <head>
```

Comment:
Nakoľko používaš diakritiku, tak by som sem odporučil prihodiť `<meta charset="UTF-8">` (v skratke: nastaví to encoding na `UTF-8`, teda aj znaky ako _"č"_, _"v"_ budú zobrazené korektne).